ignite/pkg/cosmosgen/generate.go

```diff
@@ -29,6 +30,7 @@ import (
 
 const (
 	moduleCacheNamespace       = "generate.setup.module"
+	sdkModuleCacheNamespace    = "generate.setup.sdk_module"
 	includeProtoCacheNamespace = "generator.includes.proto"
```

Comment:
Let's keep all namespaces with the same pattern
```suggestion
	includeProtoCacheNamespace = "generate.includes.proto"
```